### ISSUE TITLE
[Bug]: Id allocator not updated

### ISSUE THREAD

**Issue by BUPTAnderson** [milvus-io/milvus#20030]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus v2.1.0
- OS(Ubuntu or CentOS): CentOS
- CPU/Memory: 32core/192GB
- GPU: 
- Others:
```

### Current Behavior

`milvus/internal/rootcoord/root_coord.go`

inside tsLoop() method: [tsLoop()](https://github.com/milvus-io/milvus/blob/95f7bf44c4fce96e1766f628cf3682650760295b/internal/rootcoord/root_coord.go#L243-L249)
`tsoAllocator` was update twice, but `idAllocator` not updated.

**Comment by BUPTAnderson** [comment]
Fixed in #20029

**Comment by yanliang567** [comment]
/assign @xiaofan-luan 
please help to review the pr above
